{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "%load_ext tensorboard.notebook\n",
    "from tensorboardX import SummaryWriter"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Launching TensorBoard..."
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-5128.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXI9bG9ncy9leHB0MS8iXSwiY29uZmlndXJlX2t3YXJncyI6e30sIndvcmtpbmdfZGlyZWN0b3J5IjoiL1VzZXJzL3N1c2hhbnRhL0RvY3VtZW50cy9LYWdnbGUvSWxsaW5vaXMvQ1MgNTk4IC0gQXBwbGllZCBNYWNoaW5lIExlYXJuaW5nL0hXOCJ9', 'db': '', 'logdir': 'logs/expt1/', 'path_prefix': '', 'pid': 5128, 'port': 6008, 'start_time': 1555223021, 'version': '1.13.1'}\n",
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-5110.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXI9bG9ncy9leHB0MSJdLCJjb25maWd1cmVfa3dhcmdzIjp7fSwid29ya2luZ19kaXJlY3RvcnkiOiIvVXNlcnMvc3VzaGFudGEvRG9jdW1lbnRzL0thZ2dsZS9JbGxpbm9pcy9DUyA1OTggLSBBcHBsaWVkIE1hY2hpbmUgTGVhcm5pbmcvSFc4In0=', 'db': '', 'logdir': 'logs/expt1', 'path_prefix': '', 'pid': 5110, 'port': 6007, 'start_time': 1555222942, 'version': '1.13.1'}\n",
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-4896.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXIiLCJsb2dzL2V4cHQxIl0sImNvbmZpZ3VyZV9rd2FyZ3MiOnt9LCJ3b3JraW5nX2RpcmVjdG9yeSI6Ii9Vc2Vycy9zdXNoYW50YS9Eb2N1bWVudHMvS2FnZ2xlL0lsbGlub2lzL0NTIDU5OCAtIEFwcGxpZWQgTWFjaGluZSBMZWFybmluZy9IVzgifQ==', 'db': '', 'logdir': 'logs/expt1', 'path_prefix': '', 'pid': 4896, 'port': 6006, 'start_time': 1555222192, 'version': '1.13.1'}\n",
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-5128.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXI9bG9ncy9leHB0MS8iXSwiY29uZmlndXJlX2t3YXJncyI6e30sIndvcmtpbmdfZGlyZWN0b3J5IjoiL1VzZXJzL3N1c2hhbnRhL0RvY3VtZW50cy9LYWdnbGUvSWxsaW5vaXMvQ1MgNTk4IC0gQXBwbGllZCBNYWNoaW5lIExlYXJuaW5nL0hXOCJ9', 'db': '', 'logdir': 'logs/expt1/', 'path_prefix': '', 'pid': 5128, 'port': 6008, 'start_time': 1555223021, 'version': '1.13.1'}\n",
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-5110.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXI9bG9ncy9leHB0MSJdLCJjb25maWd1cmVfa3dhcmdzIjp7fSwid29ya2luZ19kaXJlY3RvcnkiOiIvVXNlcnMvc3VzaGFudGEvRG9jdW1lbnRzL0thZ2dsZS9JbGxpbm9pcy9DUyA1OTggLSBBcHBsaWVkIE1hY2hpbmUgTGVhcm5pbmcvSFc4In0=', 'db': '', 'logdir': 'logs/expt1', 'path_prefix': '', 'pid': 5110, 'port': 6007, 'start_time': 1555222942, 'version': '1.13.1'}\n",
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-4896.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXIiLCJsb2dzL2V4cHQxIl0sImNvbmZpZ3VyZV9rd2FyZ3MiOnt9LCJ3b3JraW5nX2RpcmVjdG9yeSI6Ii9Vc2Vycy9zdXNoYW50YS9Eb2N1bWVudHMvS2FnZ2xlL0lsbGlub2lzL0NTIDU5OCAtIEFwcGxpZWQgTWFjaGluZSBMZWFybmluZy9IVzgifQ==', 'db': '', 'logdir': 'logs/expt1', 'path_prefix': '', 'pid': 4896, 'port': 6006, 'start_time': 1555222192, 'version': '1.13.1'}\n",
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-5128.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXI9bG9ncy9leHB0MS8iXSwiY29uZmlndXJlX2t3YXJncyI6e30sIndvcmtpbmdfZGlyZWN0b3J5IjoiL1VzZXJzL3N1c2hhbnRhL0RvY3VtZW50cy9LYWdnbGUvSWxsaW5vaXMvQ1MgNTk4IC0gQXBwbGllZCBNYWNoaW5lIExlYXJuaW5nL0hXOCJ9', 'db': '', 'logdir': 'logs/expt1/', 'path_prefix': '', 'pid': 5128, 'port': 6008, 'start_time': 1555223021, 'version': '1.13.1'}\n",
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-5110.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXI9bG9ncy9leHB0MSJdLCJjb25maWd1cmVfa3dhcmdzIjp7fSwid29ya2luZ19kaXJlY3RvcnkiOiIvVXNlcnMvc3VzaGFudGEvRG9jdW1lbnRzL0thZ2dsZS9JbGxpbm9pcy9DUyA1OTggLSBBcHBsaWVkIE1hY2hpbmUgTGVhcm5pbmcvSFc4In0=', 'db': '', 'logdir': 'logs/expt1', 'path_prefix': '', 'pid': 5110, 'port': 6007, 'start_time': 1555222942, 'version': '1.13.1'}\n",
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-4896.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXIiLCJsb2dzL2V4cHQxIl0sImNvbmZpZ3VyZV9rd2FyZ3MiOnt9LCJ3b3JraW5nX2RpcmVjdG9yeSI6Ii9Vc2Vycy9zdXNoYW50YS9Eb2N1bWVudHMvS2FnZ2xlL0lsbGlub2lzL0NTIDU5OCAtIEFwcGxpZWQgTWFjaGluZSBMZWFybmluZy9IVzgifQ==', 'db': '', 'logdir': 'logs/expt1', 'path_prefix': '', 'pid': 4896, 'port': 6006, 'start_time': 1555222192, 'version': '1.13.1'}\n",
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-5128.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXI9bG9ncy9leHB0MS8iXSwiY29uZmlndXJlX2t3YXJncyI6e30sIndvcmtpbmdfZGlyZWN0b3J5IjoiL1VzZXJzL3N1c2hhbnRhL0RvY3VtZW50cy9LYWdnbGUvSWxsaW5vaXMvQ1MgNTk4IC0gQXBwbGllZCBNYWNoaW5lIExlYXJuaW5nL0hXOCJ9', 'db': '', 'logdir': 'logs/expt1/', 'path_prefix': '', 'pid': 5128, 'port': 6008, 'start_time': 1555223021, 'version': '1.13.1'}\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-7730.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXI9bG9ncy9leHB0MS8iXSwiY29uZmlndXJlX2t3YXJncyI6e30sIndvcmtpbmdfZGlyZWN0b3J5IjoiL1VzZXJzL3N1c2hhbnRhL0RvY3VtZW50cy9LYWdnbGUvSWxsaW5vaXMvQ1MgNTk4IC0gQXBwbGllZCBNYWNoaW5lIExlYXJuaW5nL0hXOCJ9', 'db': '', 'logdir': 'logs/expt1/', 'path_prefix': '', 'pid': 7730, 'port': 6009, 'start_time': 1555246117, 'version': '1.13.1'}\n",
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-5110.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXI9bG9ncy9leHB0MSJdLCJjb25maWd1cmVfa3dhcmdzIjp7fSwid29ya2luZ19kaXJlY3RvcnkiOiIvVXNlcnMvc3VzaGFudGEvRG9jdW1lbnRzL0thZ2dsZS9JbGxpbm9pcy9DUyA1OTggLSBBcHBsaWVkIE1hY2hpbmUgTGVhcm5pbmcvSFc4In0=', 'db': '', 'logdir': 'logs/expt1', 'path_prefix': '', 'pid': 5110, 'port': 6007, 'start_time': 1555222942, 'version': '1.13.1'}\n",
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-4896.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXIiLCJsb2dzL2V4cHQxIl0sImNvbmZpZ3VyZV9rd2FyZ3MiOnt9LCJ3b3JraW5nX2RpcmVjdG9yeSI6Ii9Vc2Vycy9zdXNoYW50YS9Eb2N1bWVudHMvS2FnZ2xlL0lsbGlub2lzL0NTIDU5OCAtIEFwcGxpZWQgTWFjaGluZSBMZWFybmluZy9IVzgifQ==', 'db': '', 'logdir': 'logs/expt1', 'path_prefix': '', 'pid': 4896, 'port': 6006, 'start_time': 1555222192, 'version': '1.13.1'}\n",
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-5128.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXI9bG9ncy9leHB0MS8iXSwiY29uZmlndXJlX2t3YXJncyI6e30sIndvcmtpbmdfZGlyZWN0b3J5IjoiL1VzZXJzL3N1c2hhbnRhL0RvY3VtZW50cy9LYWdnbGUvSWxsaW5vaXMvQ1MgNTk4IC0gQXBwbGllZCBNYWNoaW5lIExlYXJuaW5nL0hXOCJ9', 'db': '', 'logdir': 'logs/expt1/', 'path_prefix': '', 'pid': 5128, 'port': 6008, 'start_time': 1555223021, 'version': '1.13.1'}\n",
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-7730.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXI9bG9ncy9leHB0MS8iXSwiY29uZmlndXJlX2t3YXJncyI6e30sIndvcmtpbmdfZGlyZWN0b3J5IjoiL1VzZXJzL3N1c2hhbnRhL0RvY3VtZW50cy9LYWdnbGUvSWxsaW5vaXMvQ1MgNTk4IC0gQXBwbGllZCBNYWNoaW5lIExlYXJuaW5nL0hXOCJ9', 'db': '', 'logdir': 'logs/expt1/', 'path_prefix': '', 'pid': 7730, 'port': 6009, 'start_time': 1555246117, 'version': '1.13.1'}\n",
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-5110.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXI9bG9ncy9leHB0MSJdLCJjb25maWd1cmVfa3dhcmdzIjp7fSwid29ya2luZ19kaXJlY3RvcnkiOiIvVXNlcnMvc3VzaGFudGEvRG9jdW1lbnRzL0thZ2dsZS9JbGxpbm9pcy9DUyA1OTggLSBBcHBsaWVkIE1hY2hpbmUgTGVhcm5pbmcvSFc4In0=', 'db': '', 'logdir': 'logs/expt1', 'path_prefix': '', 'pid': 5110, 'port': 6007, 'start_time': 1555222942, 'version': '1.13.1'}\n",
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-4896.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXIiLCJsb2dzL2V4cHQxIl0sImNvbmZpZ3VyZV9rd2FyZ3MiOnt9LCJ3b3JraW5nX2RpcmVjdG9yeSI6Ii9Vc2Vycy9zdXNoYW50YS9Eb2N1bWVudHMvS2FnZ2xlL0lsbGlub2lzL0NTIDU5OCAtIEFwcGxpZWQgTWFjaGluZSBMZWFybmluZy9IVzgifQ==', 'db': '', 'logdir': 'logs/expt1', 'path_prefix': '', 'pid': 4896, 'port': 6006, 'start_time': 1555222192, 'version': '1.13.1'}\n",
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-5128.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXI9bG9ncy9leHB0MS8iXSwiY29uZmlndXJlX2t3YXJncyI6e30sIndvcmtpbmdfZGlyZWN0b3J5IjoiL1VzZXJzL3N1c2hhbnRhL0RvY3VtZW50cy9LYWdnbGUvSWxsaW5vaXMvQ1MgNTk4IC0gQXBwbGllZCBNYWNoaW5lIExlYXJuaW5nL0hXOCJ9', 'db': '', 'logdir': 'logs/expt1/', 'path_prefix': '', 'pid': 5128, 'port': 6008, 'start_time': 1555223021, 'version': '1.13.1'}\n",
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-7730.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXI9bG9ncy9leHB0MS8iXSwiY29uZmlndXJlX2t3YXJncyI6e30sIndvcmtpbmdfZGlyZWN0b3J5IjoiL1VzZXJzL3N1c2hhbnRhL0RvY3VtZW50cy9LYWdnbGUvSWxsaW5vaXMvQ1MgNTk4IC0gQXBwbGllZCBNYWNoaW5lIExlYXJuaW5nL0hXOCJ9', 'db': '', 'logdir': 'logs/expt1/', 'path_prefix': '', 'pid': 7730, 'port': 6009, 'start_time': 1555246117, 'version': '1.13.1'}\n",
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-5110.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXI9bG9ncy9leHB0MSJdLCJjb25maWd1cmVfa3dhcmdzIjp7fSwid29ya2luZ19kaXJlY3RvcnkiOiIvVXNlcnMvc3VzaGFudGEvRG9jdW1lbnRzL0thZ2dsZS9JbGxpbm9pcy9DUyA1OTggLSBBcHBsaWVkIE1hY2hpbmUgTGVhcm5pbmcvSFc4In0=', 'db': '', 'logdir': 'logs/expt1', 'path_prefix': '', 'pid': 5110, 'port': 6007, 'start_time': 1555222942, 'version': '1.13.1'}\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-4896.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXIiLCJsb2dzL2V4cHQxIl0sImNvbmZpZ3VyZV9rd2FyZ3MiOnt9LCJ3b3JraW5nX2RpcmVjdG9yeSI6Ii9Vc2Vycy9zdXNoYW50YS9Eb2N1bWVudHMvS2FnZ2xlL0lsbGlub2lzL0NTIDU5OCAtIEFwcGxpZWQgTWFjaGluZSBMZWFybmluZy9IVzgifQ==', 'db': '', 'logdir': 'logs/expt1', 'path_prefix': '', 'pid': 4896, 'port': 6006, 'start_time': 1555222192, 'version': '1.13.1'}\n",
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-5128.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXI9bG9ncy9leHB0MS8iXSwiY29uZmlndXJlX2t3YXJncyI6e30sIndvcmtpbmdfZGlyZWN0b3J5IjoiL1VzZXJzL3N1c2hhbnRhL0RvY3VtZW50cy9LYWdnbGUvSWxsaW5vaXMvQ1MgNTk4IC0gQXBwbGllZCBNYWNoaW5lIExlYXJuaW5nL0hXOCJ9', 'db': '', 'logdir': 'logs/expt1/', 'path_prefix': '', 'pid': 5128, 'port': 6008, 'start_time': 1555223021, 'version': '1.13.1'}\n",
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-7730.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXI9bG9ncy9leHB0MS8iXSwiY29uZmlndXJlX2t3YXJncyI6e30sIndvcmtpbmdfZGlyZWN0b3J5IjoiL1VzZXJzL3N1c2hhbnRhL0RvY3VtZW50cy9LYWdnbGUvSWxsaW5vaXMvQ1MgNTk4IC0gQXBwbGllZCBNYWNoaW5lIExlYXJuaW5nL0hXOCJ9', 'db': '', 'logdir': 'logs/expt1/', 'path_prefix': '', 'pid': 7730, 'port': 6009, 'start_time': 1555246117, 'version': '1.13.1'}\n",
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-5110.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXI9bG9ncy9leHB0MSJdLCJjb25maWd1cmVfa3dhcmdzIjp7fSwid29ya2luZ19kaXJlY3RvcnkiOiIvVXNlcnMvc3VzaGFudGEvRG9jdW1lbnRzL0thZ2dsZS9JbGxpbm9pcy9DUyA1OTggLSBBcHBsaWVkIE1hY2hpbmUgTGVhcm5pbmcvSFc4In0=', 'db': '', 'logdir': 'logs/expt1', 'path_prefix': '', 'pid': 5110, 'port': 6007, 'start_time': 1555222942, 'version': '1.13.1'}\n",
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-4896.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXIiLCJsb2dzL2V4cHQxIl0sImNvbmZpZ3VyZV9rd2FyZ3MiOnt9LCJ3b3JraW5nX2RpcmVjdG9yeSI6Ii9Vc2Vycy9zdXNoYW50YS9Eb2N1bWVudHMvS2FnZ2xlL0lsbGlub2lzL0NTIDU5OCAtIEFwcGxpZWQgTWFjaGluZSBMZWFybmluZy9IVzgifQ==', 'db': '', 'logdir': 'logs/expt1', 'path_prefix': '', 'pid': 4896, 'port': 6006, 'start_time': 1555222192, 'version': '1.13.1'}\n",
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-5128.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXI9bG9ncy9leHB0MS8iXSwiY29uZmlndXJlX2t3YXJncyI6e30sIndvcmtpbmdfZGlyZWN0b3J5IjoiL1VzZXJzL3N1c2hhbnRhL0RvY3VtZW50cy9LYWdnbGUvSWxsaW5vaXMvQ1MgNTk4IC0gQXBwbGllZCBNYWNoaW5lIExlYXJuaW5nL0hXOCJ9', 'db': '', 'logdir': 'logs/expt1/', 'path_prefix': '', 'pid': 5128, 'port': 6008, 'start_time': 1555223021, 'version': '1.13.1'}\n",
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-7730.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXI9bG9ncy9leHB0MS8iXSwiY29uZmlndXJlX2t3YXJncyI6e30sIndvcmtpbmdfZGlyZWN0b3J5IjoiL1VzZXJzL3N1c2hhbnRhL0RvY3VtZW50cy9LYWdnbGUvSWxsaW5vaXMvQ1MgNTk4IC0gQXBwbGllZCBNYWNoaW5lIExlYXJuaW5nL0hXOCJ9', 'db': '', 'logdir': 'logs/expt1/', 'path_prefix': '', 'pid': 7730, 'port': 6009, 'start_time': 1555246117, 'version': '1.13.1'}\n",
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-5110.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXI9bG9ncy9leHB0MSJdLCJjb25maWd1cmVfa3dhcmdzIjp7fSwid29ya2luZ19kaXJlY3RvcnkiOiIvVXNlcnMvc3VzaGFudGEvRG9jdW1lbnRzL0thZ2dsZS9JbGxpbm9pcy9DUyA1OTggLSBBcHBsaWVkIE1hY2hpbmUgTGVhcm5pbmcvSFc4In0=', 'db': '', 'logdir': 'logs/expt1', 'path_prefix': '', 'pid': 5110, 'port': 6007, 'start_time': 1555222942, 'version': '1.13.1'}\n",
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-4896.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXIiLCJsb2dzL2V4cHQxIl0sImNvbmZpZ3VyZV9rd2FyZ3MiOnt9LCJ3b3JraW5nX2RpcmVjdG9yeSI6Ii9Vc2Vycy9zdXNoYW50YS9Eb2N1bWVudHMvS2FnZ2xlL0lsbGlub2lzL0NTIDU5OCAtIEFwcGxpZWQgTWFjaGluZSBMZWFybmluZy9IVzgifQ==', 'db': '', 'logdir': 'logs/expt1', 'path_prefix': '', 'pid': 4896, 'port': 6006, 'start_time': 1555222192, 'version': '1.13.1'}\n",
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-5128.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXI9bG9ncy9leHB0MS8iXSwiY29uZmlndXJlX2t3YXJncyI6e30sIndvcmtpbmdfZGlyZWN0b3J5IjoiL1VzZXJzL3N1c2hhbnRhL0RvY3VtZW50cy9LYWdnbGUvSWxsaW5vaXMvQ1MgNTk4IC0gQXBwbGllZCBNYWNoaW5lIExlYXJuaW5nL0hXOCJ9', 'db': '', 'logdir': 'logs/expt1/', 'path_prefix': '', 'pid': 5128, 'port': 6008, 'start_time': 1555223021, 'version': '1.13.1'}\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-7730.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXI9bG9ncy9leHB0MS8iXSwiY29uZmlndXJlX2t3YXJncyI6e30sIndvcmtpbmdfZGlyZWN0b3J5IjoiL1VzZXJzL3N1c2hhbnRhL0RvY3VtZW50cy9LYWdnbGUvSWxsaW5vaXMvQ1MgNTk4IC0gQXBwbGllZCBNYWNoaW5lIExlYXJuaW5nL0hXOCJ9', 'db': '', 'logdir': 'logs/expt1/', 'path_prefix': '', 'pid': 7730, 'port': 6009, 'start_time': 1555246117, 'version': '1.13.1'}\n",
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-5110.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXI9bG9ncy9leHB0MSJdLCJjb25maWd1cmVfa3dhcmdzIjp7fSwid29ya2luZ19kaXJlY3RvcnkiOiIvVXNlcnMvc3VzaGFudGEvRG9jdW1lbnRzL0thZ2dsZS9JbGxpbm9pcy9DUyA1OTggLSBBcHBsaWVkIE1hY2hpbmUgTGVhcm5pbmcvSFc4In0=', 'db': '', 'logdir': 'logs/expt1', 'path_prefix': '', 'pid': 5110, 'port': 6007, 'start_time': 1555222942, 'version': '1.13.1'}\n",
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-4896.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXIiLCJsb2dzL2V4cHQxIl0sImNvbmZpZ3VyZV9rd2FyZ3MiOnt9LCJ3b3JraW5nX2RpcmVjdG9yeSI6Ii9Vc2Vycy9zdXNoYW50YS9Eb2N1bWVudHMvS2FnZ2xlL0lsbGlub2lzL0NTIDU5OCAtIEFwcGxpZWQgTWFjaGluZSBMZWFybmluZy9IVzgifQ==', 'db': '', 'logdir': 'logs/expt1', 'path_prefix': '', 'pid': 4896, 'port': 6006, 'start_time': 1555222192, 'version': '1.13.1'}\n",
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-5128.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXI9bG9ncy9leHB0MS8iXSwiY29uZmlndXJlX2t3YXJncyI6e30sIndvcmtpbmdfZGlyZWN0b3J5IjoiL1VzZXJzL3N1c2hhbnRhL0RvY3VtZW50cy9LYWdnbGUvSWxsaW5vaXMvQ1MgNTk4IC0gQXBwbGllZCBNYWNoaW5lIExlYXJuaW5nL0hXOCJ9', 'db': '', 'logdir': 'logs/expt1/', 'path_prefix': '', 'pid': 5128, 'port': 6008, 'start_time': 1555223021, 'version': '1.13.1'}\n",
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-7730.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXI9bG9ncy9leHB0MS8iXSwiY29uZmlndXJlX2t3YXJncyI6e30sIndvcmtpbmdfZGlyZWN0b3J5IjoiL1VzZXJzL3N1c2hhbnRhL0RvY3VtZW50cy9LYWdnbGUvSWxsaW5vaXMvQ1MgNTk4IC0gQXBwbGllZCBNYWNoaW5lIExlYXJuaW5nL0hXOCJ9', 'db': '', 'logdir': 'logs/expt1/', 'path_prefix': '', 'pid': 7730, 'port': 6009, 'start_time': 1555246117, 'version': '1.13.1'}\n",
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-5110.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXI9bG9ncy9leHB0MSJdLCJjb25maWd1cmVfa3dhcmdzIjp7fSwid29ya2luZ19kaXJlY3RvcnkiOiIvVXNlcnMvc3VzaGFudGEvRG9jdW1lbnRzL0thZ2dsZS9JbGxpbm9pcy9DUyA1OTggLSBBcHBsaWVkIE1hY2hpbmUgTGVhcm5pbmcvSFc4In0=', 'db': '', 'logdir': 'logs/expt1', 'path_prefix': '', 'pid': 5110, 'port': 6007, 'start_time': 1555222942, 'version': '1.13.1'}\n",
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-4896.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXIiLCJsb2dzL2V4cHQxIl0sImNvbmZpZ3VyZV9rd2FyZ3MiOnt9LCJ3b3JraW5nX2RpcmVjdG9yeSI6Ii9Vc2Vycy9zdXNoYW50YS9Eb2N1bWVudHMvS2FnZ2xlL0lsbGlub2lzL0NTIDU5OCAtIEFwcGxpZWQgTWFjaGluZSBMZWFybmluZy9IVzgifQ==', 'db': '', 'logdir': 'logs/expt1', 'path_prefix': '', 'pid': 4896, 'port': 6006, 'start_time': 1555222192, 'version': '1.13.1'}\n",
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-5128.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXI9bG9ncy9leHB0MS8iXSwiY29uZmlndXJlX2t3YXJncyI6e30sIndvcmtpbmdfZGlyZWN0b3J5IjoiL1VzZXJzL3N1c2hhbnRhL0RvY3VtZW50cy9LYWdnbGUvSWxsaW5vaXMvQ1MgNTk4IC0gQXBwbGllZCBNYWNoaW5lIExlYXJuaW5nL0hXOCJ9', 'db': '', 'logdir': 'logs/expt1/', 'path_prefix': '', 'pid': 5128, 'port': 6008, 'start_time': 1555223021, 'version': '1.13.1'}\n",
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-7730.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXI9bG9ncy9leHB0MS8iXSwiY29uZmlndXJlX2t3YXJncyI6e30sIndvcmtpbmdfZGlyZWN0b3J5IjoiL1VzZXJzL3N1c2hhbnRhL0RvY3VtZW50cy9LYWdnbGUvSWxsaW5vaXMvQ1MgNTk4IC0gQXBwbGllZCBNYWNoaW5lIExlYXJuaW5nL0hXOCJ9', 'db': '', 'logdir': 'logs/expt1/', 'path_prefix': '', 'pid': 7730, 'port': 6009, 'start_time': 1555246117, 'version': '1.13.1'}\n",
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-5110.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXI9bG9ncy9leHB0MSJdLCJjb25maWd1cmVfa3dhcmdzIjp7fSwid29ya2luZ19kaXJlY3RvcnkiOiIvVXNlcnMvc3VzaGFudGEvRG9jdW1lbnRzL0thZ2dsZS9JbGxpbm9pcy9DUyA1OTggLSBBcHBsaWVkIE1hY2hpbmUgTGVhcm5pbmcvSFc4In0=', 'db': '', 'logdir': 'logs/expt1', 'path_prefix': '', 'pid': 5110, 'port': 6007, 'start_time': 1555222942, 'version': '1.13.1'}\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-4896.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXIiLCJsb2dzL2V4cHQxIl0sImNvbmZpZ3VyZV9rd2FyZ3MiOnt9LCJ3b3JraW5nX2RpcmVjdG9yeSI6Ii9Vc2Vycy9zdXNoYW50YS9Eb2N1bWVudHMvS2FnZ2xlL0lsbGlub2lzL0NTIDU5OCAtIEFwcGxpZWQgTWFjaGluZSBMZWFybmluZy9IVzgifQ==', 'db': '', 'logdir': 'logs/expt1', 'path_prefix': '', 'pid': 4896, 'port': 6006, 'start_time': 1555222192, 'version': '1.13.1'}\n",
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-5128.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXI9bG9ncy9leHB0MS8iXSwiY29uZmlndXJlX2t3YXJncyI6e30sIndvcmtpbmdfZGlyZWN0b3J5IjoiL1VzZXJzL3N1c2hhbnRhL0RvY3VtZW50cy9LYWdnbGUvSWxsaW5vaXMvQ1MgNTk4IC0gQXBwbGllZCBNYWNoaW5lIExlYXJuaW5nL0hXOCJ9', 'db': '', 'logdir': 'logs/expt1/', 'path_prefix': '', 'pid': 5128, 'port': 6008, 'start_time': 1555223021, 'version': '1.13.1'}\n",
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-7730.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXI9bG9ncy9leHB0MS8iXSwiY29uZmlndXJlX2t3YXJncyI6e30sIndvcmtpbmdfZGlyZWN0b3J5IjoiL1VzZXJzL3N1c2hhbnRhL0RvY3VtZW50cy9LYWdnbGUvSWxsaW5vaXMvQ1MgNTk4IC0gQXBwbGllZCBNYWNoaW5lIExlYXJuaW5nL0hXOCJ9', 'db': '', 'logdir': 'logs/expt1/', 'path_prefix': '', 'pid': 7730, 'port': 6009, 'start_time': 1555246117, 'version': '1.13.1'}\n",
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-5110.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXI9bG9ncy9leHB0MSJdLCJjb25maWd1cmVfa3dhcmdzIjp7fSwid29ya2luZ19kaXJlY3RvcnkiOiIvVXNlcnMvc3VzaGFudGEvRG9jdW1lbnRzL0thZ2dsZS9JbGxpbm9pcy9DUyA1OTggLSBBcHBsaWVkIE1hY2hpbmUgTGVhcm5pbmcvSFc4In0=', 'db': '', 'logdir': 'logs/expt1', 'path_prefix': '', 'pid': 5110, 'port': 6007, 'start_time': 1555222942, 'version': '1.13.1'}\n",
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-4896.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXIiLCJsb2dzL2V4cHQxIl0sImNvbmZpZ3VyZV9rd2FyZ3MiOnt9LCJ3b3JraW5nX2RpcmVjdG9yeSI6Ii9Vc2Vycy9zdXNoYW50YS9Eb2N1bWVudHMvS2FnZ2xlL0lsbGlub2lzL0NTIDU5OCAtIEFwcGxpZWQgTWFjaGluZSBMZWFybmluZy9IVzgifQ==', 'db': '', 'logdir': 'logs/expt1', 'path_prefix': '', 'pid': 4896, 'port': 6006, 'start_time': 1555222192, 'version': '1.13.1'}\n",
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-5128.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXI9bG9ncy9leHB0MS8iXSwiY29uZmlndXJlX2t3YXJncyI6e30sIndvcmtpbmdfZGlyZWN0b3J5IjoiL1VzZXJzL3N1c2hhbnRhL0RvY3VtZW50cy9LYWdnbGUvSWxsaW5vaXMvQ1MgNTk4IC0gQXBwbGllZCBNYWNoaW5lIExlYXJuaW5nL0hXOCJ9', 'db': '', 'logdir': 'logs/expt1/', 'path_prefix': '', 'pid': 5128, 'port': 6008, 'start_time': 1555223021, 'version': '1.13.1'}\n",
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-7730.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXI9bG9ncy9leHB0MS8iXSwiY29uZmlndXJlX2t3YXJncyI6e30sIndvcmtpbmdfZGlyZWN0b3J5IjoiL1VzZXJzL3N1c2hhbnRhL0RvY3VtZW50cy9LYWdnbGUvSWxsaW5vaXMvQ1MgNTk4IC0gQXBwbGllZCBNYWNoaW5lIExlYXJuaW5nL0hXOCJ9', 'db': '', 'logdir': 'logs/expt1/', 'path_prefix': '', 'pid': 7730, 'port': 6009, 'start_time': 1555246117, 'version': '1.13.1'}\n",
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-5110.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXI9bG9ncy9leHB0MSJdLCJjb25maWd1cmVfa3dhcmdzIjp7fSwid29ya2luZ19kaXJlY3RvcnkiOiIvVXNlcnMvc3VzaGFudGEvRG9jdW1lbnRzL0thZ2dsZS9JbGxpbm9pcy9DUyA1OTggLSBBcHBsaWVkIE1hY2hpbmUgTGVhcm5pbmcvSFc4In0=', 'db': '', 'logdir': 'logs/expt1', 'path_prefix': '', 'pid': 5110, 'port': 6007, 'start_time': 1555222942, 'version': '1.13.1'}\n",
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-4896.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXIiLCJsb2dzL2V4cHQxIl0sImNvbmZpZ3VyZV9rd2FyZ3MiOnt9LCJ3b3JraW5nX2RpcmVjdG9yeSI6Ii9Vc2Vycy9zdXNoYW50YS9Eb2N1bWVudHMvS2FnZ2xlL0lsbGlub2lzL0NTIDU5OCAtIEFwcGxpZWQgTWFjaGluZSBMZWFybmluZy9IVzgifQ==', 'db': '', 'logdir': 'logs/expt1', 'path_prefix': '', 'pid': 4896, 'port': 6006, 'start_time': 1555222192, 'version': '1.13.1'}\n",
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-5128.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXI9bG9ncy9leHB0MS8iXSwiY29uZmlndXJlX2t3YXJncyI6e30sIndvcmtpbmdfZGlyZWN0b3J5IjoiL1VzZXJzL3N1c2hhbnRhL0RvY3VtZW50cy9LYWdnbGUvSWxsaW5vaXMvQ1MgNTk4IC0gQXBwbGllZCBNYWNoaW5lIExlYXJuaW5nL0hXOCJ9', 'db': '', 'logdir': 'logs/expt1/', 'path_prefix': '', 'pid': 5128, 'port': 6008, 'start_time': 1555223021, 'version': '1.13.1'}\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-7730.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXI9bG9ncy9leHB0MS8iXSwiY29uZmlndXJlX2t3YXJncyI6e30sIndvcmtpbmdfZGlyZWN0b3J5IjoiL1VzZXJzL3N1c2hhbnRhL0RvY3VtZW50cy9LYWdnbGUvSWxsaW5vaXMvQ1MgNTk4IC0gQXBwbGllZCBNYWNoaW5lIExlYXJuaW5nL0hXOCJ9', 'db': '', 'logdir': 'logs/expt1/', 'path_prefix': '', 'pid': 7730, 'port': 6009, 'start_time': 1555246117, 'version': '1.13.1'}\n",
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-5110.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXI9bG9ncy9leHB0MSJdLCJjb25maWd1cmVfa3dhcmdzIjp7fSwid29ya2luZ19kaXJlY3RvcnkiOiIvVXNlcnMvc3VzaGFudGEvRG9jdW1lbnRzL0thZ2dsZS9JbGxpbm9pcy9DUyA1OTggLSBBcHBsaWVkIE1hY2hpbmUgTGVhcm5pbmcvSFc4In0=', 'db': '', 'logdir': 'logs/expt1', 'path_prefix': '', 'pid': 5110, 'port': 6007, 'start_time': 1555222942, 'version': '1.13.1'}\n",
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-4896.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXIiLCJsb2dzL2V4cHQxIl0sImNvbmZpZ3VyZV9rd2FyZ3MiOnt9LCJ3b3JraW5nX2RpcmVjdG9yeSI6Ii9Vc2Vycy9zdXNoYW50YS9Eb2N1bWVudHMvS2FnZ2xlL0lsbGlub2lzL0NTIDU5OCAtIEFwcGxpZWQgTWFjaGluZSBMZWFybmluZy9IVzgifQ==', 'db': '', 'logdir': 'logs/expt1', 'path_prefix': '', 'pid': 4896, 'port': 6006, 'start_time': 1555222192, 'version': '1.13.1'}\n",
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-5128.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXI9bG9ncy9leHB0MS8iXSwiY29uZmlndXJlX2t3YXJncyI6e30sIndvcmtpbmdfZGlyZWN0b3J5IjoiL1VzZXJzL3N1c2hhbnRhL0RvY3VtZW50cy9LYWdnbGUvSWxsaW5vaXMvQ1MgNTk4IC0gQXBwbGllZCBNYWNoaW5lIExlYXJuaW5nL0hXOCJ9', 'db': '', 'logdir': 'logs/expt1/', 'path_prefix': '', 'pid': 5128, 'port': 6008, 'start_time': 1555223021, 'version': '1.13.1'}\n",
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-7730.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXI9bG9ncy9leHB0MS8iXSwiY29uZmlndXJlX2t3YXJncyI6e30sIndvcmtpbmdfZGlyZWN0b3J5IjoiL1VzZXJzL3N1c2hhbnRhL0RvY3VtZW50cy9LYWdnbGUvSWxsaW5vaXMvQ1MgNTk4IC0gQXBwbGllZCBNYWNoaW5lIExlYXJuaW5nL0hXOCJ9', 'db': '', 'logdir': 'logs/expt1/', 'path_prefix': '', 'pid': 7730, 'port': 6009, 'start_time': 1555246117, 'version': '1.13.1'}\n",
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-5110.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXI9bG9ncy9leHB0MSJdLCJjb25maWd1cmVfa3dhcmdzIjp7fSwid29ya2luZ19kaXJlY3RvcnkiOiIvVXNlcnMvc3VzaGFudGEvRG9jdW1lbnRzL0thZ2dsZS9JbGxpbm9pcy9DUyA1OTggLSBBcHBsaWVkIE1hY2hpbmUgTGVhcm5pbmcvSFc4In0=', 'db': '', 'logdir': 'logs/expt1', 'path_prefix': '', 'pid': 5110, 'port': 6007, 'start_time': 1555222942, 'version': '1.13.1'}\n",
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-4896.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXIiLCJsb2dzL2V4cHQxIl0sImNvbmZpZ3VyZV9rd2FyZ3MiOnt9LCJ3b3JraW5nX2RpcmVjdG9yeSI6Ii9Vc2Vycy9zdXNoYW50YS9Eb2N1bWVudHMvS2FnZ2xlL0lsbGlub2lzL0NTIDU5OCAtIEFwcGxpZWQgTWFjaGluZSBMZWFybmluZy9IVzgifQ==', 'db': '', 'logdir': 'logs/expt1', 'path_prefix': '', 'pid': 4896, 'port': 6006, 'start_time': 1555222192, 'version': '1.13.1'}\n",
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-5128.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXI9bG9ncy9leHB0MS8iXSwiY29uZmlndXJlX2t3YXJncyI6e30sIndvcmtpbmdfZGlyZWN0b3J5IjoiL1VzZXJzL3N1c2hhbnRhL0RvY3VtZW50cy9LYWdnbGUvSWxsaW5vaXMvQ1MgNTk4IC0gQXBwbGllZCBNYWNoaW5lIExlYXJuaW5nL0hXOCJ9', 'db': '', 'logdir': 'logs/expt1/', 'path_prefix': '', 'pid': 5128, 'port': 6008, 'start_time': 1555223021, 'version': '1.13.1'}\n",
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-7730.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXI9bG9ncy9leHB0MS8iXSwiY29uZmlndXJlX2t3YXJncyI6e30sIndvcmtpbmdfZGlyZWN0b3J5IjoiL1VzZXJzL3N1c2hhbnRhL0RvY3VtZW50cy9LYWdnbGUvSWxsaW5vaXMvQ1MgNTk4IC0gQXBwbGllZCBNYWNoaW5lIExlYXJuaW5nL0hXOCJ9', 'db': '', 'logdir': 'logs/expt1/', 'path_prefix': '', 'pid': 7730, 'port': 6009, 'start_time': 1555246117, 'version': '1.13.1'}\n",
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-5110.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXI9bG9ncy9leHB0MSJdLCJjb25maWd1cmVfa3dhcmdzIjp7fSwid29ya2luZ19kaXJlY3RvcnkiOiIvVXNlcnMvc3VzaGFudGEvRG9jdW1lbnRzL0thZ2dsZS9JbGxpbm9pcy9DUyA1OTggLSBBcHBsaWVkIE1hY2hpbmUgTGVhcm5pbmcvSFc4In0=', 'db': '', 'logdir': 'logs/expt1', 'path_prefix': '', 'pid': 5110, 'port': 6007, 'start_time': 1555222942, 'version': '1.13.1'}\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-4896.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXIiLCJsb2dzL2V4cHQxIl0sImNvbmZpZ3VyZV9rd2FyZ3MiOnt9LCJ3b3JraW5nX2RpcmVjdG9yeSI6Ii9Vc2Vycy9zdXNoYW50YS9Eb2N1bWVudHMvS2FnZ2xlL0lsbGlub2lzL0NTIDU5OCAtIEFwcGxpZWQgTWFjaGluZSBMZWFybmluZy9IVzgifQ==', 'db': '', 'logdir': 'logs/expt1', 'path_prefix': '', 'pid': 4896, 'port': 6006, 'start_time': 1555222192, 'version': '1.13.1'}\n",
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-5128.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXI9bG9ncy9leHB0MS8iXSwiY29uZmlndXJlX2t3YXJncyI6e30sIndvcmtpbmdfZGlyZWN0b3J5IjoiL1VzZXJzL3N1c2hhbnRhL0RvY3VtZW50cy9LYWdnbGUvSWxsaW5vaXMvQ1MgNTk4IC0gQXBwbGllZCBNYWNoaW5lIExlYXJuaW5nL0hXOCJ9', 'db': '', 'logdir': 'logs/expt1/', 'path_prefix': '', 'pid': 5128, 'port': 6008, 'start_time': 1555223021, 'version': '1.13.1'}\n",
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-7730.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXI9bG9ncy9leHB0MS8iXSwiY29uZmlndXJlX2t3YXJncyI6e30sIndvcmtpbmdfZGlyZWN0b3J5IjoiL1VzZXJzL3N1c2hhbnRhL0RvY3VtZW50cy9LYWdnbGUvSWxsaW5vaXMvQ1MgNTk4IC0gQXBwbGllZCBNYWNoaW5lIExlYXJuaW5nL0hXOCJ9', 'db': '', 'logdir': 'logs/expt1/', 'path_prefix': '', 'pid': 7730, 'port': 6009, 'start_time': 1555246117, 'version': '1.13.1'}\n",
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-5110.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXI9bG9ncy9leHB0MSJdLCJjb25maWd1cmVfa3dhcmdzIjp7fSwid29ya2luZ19kaXJlY3RvcnkiOiIvVXNlcnMvc3VzaGFudGEvRG9jdW1lbnRzL0thZ2dsZS9JbGxpbm9pcy9DUyA1OTggLSBBcHBsaWVkIE1hY2hpbmUgTGVhcm5pbmcvSFc4In0=', 'db': '', 'logdir': 'logs/expt1', 'path_prefix': '', 'pid': 5110, 'port': 6007, 'start_time': 1555222942, 'version': '1.13.1'}\n",
      "invalid info file: '/var/folders/w7/wq6tmkfx393_lw9pndw50fbc0000gn/T/.tensorboard-info/pid-4896.info'\n",
      "Traceback (most recent call last):\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 317, in get_all\n",
      "    info = _info_from_string(contents)\n",
      "  File \"/Users/sushanta/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\", line 158, in _info_from_string\n",
      "    raise ValueError(\"incompatible version: %r\" % (json_value,))\n",
      "ValueError: incompatible version: {'cache_key': 'eyJhcmd1bWVudHMiOlsiLS1sb2dkaXIiLCJsb2dzL2V4cHQxIl0sImNvbmZpZ3VyZV9rd2FyZ3MiOnt9LCJ3b3JraW5nX2RpcmVjdG9yeSI6Ii9Vc2Vycy9zdXNoYW50YS9Eb2N1bWVudHMvS2FnZ2xlL0lsbGlub2lzL0NTIDU5OCAtIEFwcGxpZWQgTWFjaGluZSBMZWFybmluZy9IVzgifQ==', 'db': '', 'logdir': 'logs/expt1', 'path_prefix': '', 'pid': 4896, 'port': 6006, 'start_time': 1555222192, 'version': '1.13.1'}\n"
     ]
    },
    {
     "ename": "KeyboardInterrupt",
     "evalue": "",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mKeyboardInterrupt\u001b[0m                         Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-2-ef5a140db0d1>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      1\u001b[0m \u001b[0;31m#%tensorboard --logdir=logs\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 2\u001b[0;31m \u001b[0mget_ipython\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrun_line_magic\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m'tensorboard'\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m'--logdir=logs/expt1/'\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
      "\u001b[0;32m~/anaconda3/lib/python3.6/site-packages/IPython/core/interactiveshell.py\u001b[0m in \u001b[0;36mrun_line_magic\u001b[0;34m(self, magic_name, line, _stack_depth)\u001b[0m\n\u001b[1;32m   2129\u001b[0m                 \u001b[0mkwargs\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m'local_ns'\u001b[0m\u001b[0;34m]\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0msys\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_getframe\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mstack_depth\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mf_locals\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   2130\u001b[0m             \u001b[0;32mwith\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mbuiltin_trap\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m-> 2131\u001b[0;31m                 \u001b[0mresult\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mfn\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m*\u001b[0m\u001b[0margs\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m**\u001b[0m\u001b[0mkwargs\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m   2132\u001b[0m             \u001b[0;32mreturn\u001b[0m \u001b[0mresult\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   2133\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m~/anaconda3/lib/python3.6/site-packages/tensorboard/notebook.py\u001b[0m in \u001b[0;36m_start_magic\u001b[0;34m(line)\u001b[0m\n\u001b[1;32m    100\u001b[0m \u001b[0;32mdef\u001b[0m \u001b[0m_start_magic\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mline\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    101\u001b[0m   \u001b[0;34m\"\"\"Implementation of the `%tensorboard` line magic.\"\"\"\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 102\u001b[0;31m   \u001b[0;32mreturn\u001b[0m \u001b[0mstart\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mline\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    103\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    104\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m~/anaconda3/lib/python3.6/site-packages/tensorboard/notebook.py\u001b[0m in \u001b[0;36mstart\u001b[0;34m(args_string)\u001b[0m\n\u001b[1;32m    135\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    136\u001b[0m   \u001b[0mparsed_args\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mshlex\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msplit\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0margs_string\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mcomments\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;32mTrue\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mposix\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;32mTrue\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 137\u001b[0;31m   \u001b[0mstart_result\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mmanager\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mstart\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mparsed_args\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    138\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    139\u001b[0m   \u001b[0;32mif\u001b[0m \u001b[0misinstance\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mstart_result\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mmanager\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mStartLaunched\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m~/anaconda3/lib/python3.6/site-packages/tensorboard/manager.py\u001b[0m in \u001b[0;36mstart\u001b[0;34m(arguments, timeout)\u001b[0m\n\u001b[1;32m    412\u001b[0m   \u001b[0mend_time\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mstart_time\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0mtimeout\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    413\u001b[0m   \u001b[0;32mwhile\u001b[0m \u001b[0mdatetime\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mdatetime\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mnow\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m \u001b[0;34m<\u001b[0m \u001b[0mend_time\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 414\u001b[0;31m     \u001b[0mtime\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msleep\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mpoll_interval_seconds\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    415\u001b[0m     \u001b[0msubprocess_result\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mp\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mpoll\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    416\u001b[0m     \u001b[0;32mif\u001b[0m \u001b[0msubprocess_result\u001b[0m \u001b[0;32mis\u001b[0m \u001b[0;32mnot\u001b[0m \u001b[0;32mNone\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;31mKeyboardInterrupt\u001b[0m: "
     ]
    }
   ],
   "source": [
    "#%tensorboard --logdir=logs\n",
    "%tensorboard --logdir=logs/expt1/"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.5"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
